test-projects/test-hidden-file/test/asset-compute/worker/.test

```diff
@@ -0,0 +1 @@
+hello
```

Comment:
You could have "Bad guy" in here ;-)

Reply:
Wait, shouldn't this be one level up, inside the `asset-compute` folder? As the original error is when that includes an extra file or directory that is not an action inside the manifest.yml?

And maybe use `.DS_Store` as name to closely reproduce the original issue.

Reply:
oops yeah meant to put this one level up. and I'll name it .DS_Store so it makes more sense